src/app/ui/NewTaskForm.tsx

```diff
@@ -243,13 +245,14 @@ const NewTaskFormInputs = () => {
         <StyledTextField
           type="text"
           padding="8px 0px"
+          autoFocus={true}
           value={title}
           onChange={(e) => store.dispatch(setCreateTaskFields({ targetField: 'title', value: e.target.value }))}
         />
       </Stack>
       <Stack direction="column" rowGap={1} m="16px 0px">
         <Typography variant="md">Description</Typography>
-        <TapWriteTaskEditor
+        <TapWriteTaskDescription
```

Comment:
Let's remove the styled component for Tapwrite from everywhere and utilize the `editorClass` prop. Let's not style font size directly from the styled components instead lets utilize css selectors such as `.tiptap p`, `.tiptap h1` wherever required.